### ISSUE TITLE
Workspace's service quotas accessible through UI and CLI

### ISSUE THREAD

**Issue by robert-becker-hs** [aws/amazon-managed-service-for-prometheus-roadmap#20]
It could be advantageous for some customers to have visibility into what the current service quotas are, per workspace. This would especially come in handy if there are multiple workspaces with different data setups. I could imagine this being part of the workspace UI and/or accessible through the AWS CLI.

**Comment by ampabhi-aws** [comment]
Yup, agreed that we should expose these quotas in the AMP console. Would you also like a dedicated API for quotas? Or more just a CLI that integrates with the existing Service Quotas?
